src/main/java/gift/service/MemberService.java

```diff
@@ -15,12 +15,16 @@ public MemberService(MemberRepository memberRepository) {
         this.memberRepository = memberRepository;
     }
 
-    public LoginMemberDto selectLoginMemberById(Long id) {
-        Member member = memberRepository.findById(id).get();
-        return new LoginMemberDto(member.getId(), member.getName(), member.getEmail(),
-            member.getRole());
+    @Transactional(readOnly = true)
+    public LoginMemberDto selectLoginMemberById(Long id) throws IllegalArgumentException {
```

Comment:
Unchecked Exception은 굳이 적지 않는 편입니다. (하면 안되는건 아니에요)
```suggestion
    public LoginMemberDto selectLoginMemberById(Long id) {
```